tests/functional/v2/server/testListMetrics.js

```diff
@@ -30,10 +30,15 @@ const emptyOperationsResponse = Object.values(operationToResponse)
 
 async function listMetrics(level, resources, start, end, force403 = false) {
     const body = {
-        timeRange: [start, end],
         [level]: resources,
     };
 
+    if (end !== undefined) {
+        body.timeRange = [start, end];
+    } else {
+        body.timeRange = [start];
+    }
```

Comment:
I think `[start, end]` works in both cases as end would be undefined.

Reply:
it serializes to `[0, null]` for some reason if you leave it in